data/core/units/monsters/Fungoid_Body.cfg

```diff
@@ -0,0 +1,275 @@
+#textdomain wesnoth-units
+
+[unit_type]
+    id=Fungoid Body
+    name= _ "Fungoid Body"
+    race=monster
+    image=units/monsters/core-compact.png
+    image_icon="units/monsters/core-compact.png~SCALE(72,72)"
+    [standing_anim]
+        [frame]
+            duration=100
+            image="units/monsters/core-compact.png"
+            auto_hflip=no
+        [/frame]
+    [/standing_anim]
+    [resistance]
+        blade=110
+        pierce=110
+        impact=80
+        fire=150
+        cold=150
+        arcane=200
+    [/resistance]
+    hitpoints=180
+    movement_type=smallfoot
+    movement=0
+    experience=150
+    {AMLA_DEFAULT}
+    level=3
+    alignment=chaotic
+    advances_to=null
+    cost=403
+    usage=mixed fighter
+    description= _ "This thing is impossible to describe, no one has seen anything like it before."
```

Comment:
This is like the worst possible description. I'd prefer a blank description to be filled in later over this monstrosity.

Reply:
This is taken directly from UtBS, which generally has the worst possible writing quality. My plan was and is to rewrite the description(s) once the unit(s) design has solidified.